opm/simulators/utils/ParallelRestart.hpp

```diff
@@ -34,6 +34,7 @@
 #include <optional>
 #include <map>
 #include <set>
+#include <string>
```

Comment:
Technically you need `<typeinfo>` too, if you want to invoke member functions on the object returned from `typeid`.